packages/docs/developer-resources/contractkit/reference/classes/_address_registry_.addressregistry.md

```diff
@@ -0,0 +1,64 @@
+# Class: AddressRegistry
+
+Celo Core Contract's Address Registry
+
+## Hierarchy
+
+* **AddressRegistry**
+
+## Index
+
+### Constructors
+
+* [constructor](_address_registry_.addressregistry.md#constructor)
+
+### Methods
+
+* [addressFor](_address_registry_.addressregistry.md#addressfor)
+* [allAddresses](_address_registry_.addressregistry.md#alladdresses)
+
+## Constructors
+
+###  constructor
+
+\+ **new AddressRegistry**(`kit`: [ContractKit](_kit_.contractkit.md)): *[AddressRegistry](_address_registry_.addressregistry.md)*
+
+*Defined in [packages/contractkit/src/address-registry.ts:18](https://github.com/celo-org/celo-monorepo/blob/06adf8b7a/packages/contractkit/src/address-registry.ts#L18)*
```

Comment:
the blob hash changes every commit so the CI `git diff` check doesn't work exactly like the CLI one

Reply:
@timmoreton